tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -4189,6 +4212,7 @@ declare namespace ts {
     }
     export type FileWatcherCallback = (fileName: string, eventKind: FileWatcherEventKind, modifiedTime?: Date) => void;
     export type DirectoryWatcherCallback = (fileName: string) => void;
+    export type BufferEncoding = "ascii" | "utf8" | "utf-8" | "utf16le" | "ucs2" | "ucs-2" | "base64" | "latin1" | "binary" | "hex";
```

Comment:
Kind of wish this didn't need to be exposed. Especially at the top level.

Reply:
I could undo this bit, but that would unfortunately force everyone to cast the Node function to one in our existing API which accepts any string. Of course, that's feasibly what external users have to do now, too. We just don't experience it in our repo thanks to the lack of SFT.